annlite/index.py

```diff
@@ -634,12 +634,9 @@ def dump_model(self):
 
     def dump_index(self):
         logger.info(f'Save the indexer to {self.index_path}')
-
         try:
             self.index_path.mkdir(parents=True)
 
-            self.meta_table.dump(self.index_path / 'meta_table.db')
-
             for cell_id in range(self.n_cells):
```

Comment:
The `self.meta_table.load(self.snapshot_path / 'meta_table.db')` bellow is not needed any more.

Reply:
And the the `def insert(...)` in `container.py` should also been refactored when `only_index=True`